config.toml

```
@@ -115,13 +115,13 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.18"
+latest = "v1.19"
 
-fullversion = "v1.18.0"
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "master"
-docsbranch = "master"
-deprecated = false
+githubbranch = "v1.18.8"
+docsbranch = "release-1.18"
```

This doesn't look right to me. I think this should be:
```suggestion
fullversion = "v1.19.0"
version = "v1.19"
githubbranch = "master"
docsbranch = "master"
deprecated = false
```